docs/testing-plan.md

```diff
@@ -0,0 +1,36 @@
+# Testing plan
```

Comment:
1. There's also a paper about this: https://paper.dropbox.com/doc/Testing-plan--A38aCEu1lx5TW0hIVX~OLwsIAg-2NHOfYnIOmBUQxUYOYDPF
2. We should get a good routine for writing tests on all (related) levels for each task. Is this the place to describe that kind of guidelines? Or in the Development flow paper? Or in the Testing plan paper?

For me it's ok to have this kind of docs here in the `docs/` folder. And good that you wrote this!

Reply:
I think we could develop it in paper but then document the outcome here.